AirCasting/SessionViews/ChartViewModel.swift

```diff
@@ -76,11 +77,11 @@ final class ChartViewModel: ObservableObject {
             if let average = average {
                 entries.append(ChartDataEntry(x: Double(i), y: average))
             }
+            chartStartTime = intervalEnd
```

Comment:
`chartStartTime` is a `@Published` property - are you sure we want to update it that often?